tests/integration/linux_bridge_test.py

```diff
@@ -105,6 +105,41 @@ def test_create_and_remove_linux_bridge_with_two_ports(port0_up, port1_up):
     assertlib.assert_absent(bridge_name)
 
 
+def test_remove_bridge_and_keep_slave_up(bridge0_with_port0, port0_up):
+    bridge_name = bridge0_with_port0[Interface.KEY][0][Interface.NAME]
+    port_name = port0_up[Interface.KEY][0][Interface.NAME]
+    desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: bridge_name,
+                Interface.STATE: InterfaceState.ABSENT,
+            },
+            {
+                Interface.NAME: port_name,
+                Interface.STATE: InterfaceState.UP,
+                Interface.IPV4: {'enabled': False},
+                Interface.IPV6: {'enabled': False},
+            },
```

Comment:
You can reuse `port_desired_state` here. Not a must.

Reply:
Right, thanks.